DigitalLearningSolutions.Data/Services/CentreCustomPromptsService.cs

```diff
@@ -280,5 +322,26 @@ DelegateUser delegateUser
 
             return list;
         }
+
+        public static CustomPrompt? PopulateCustomPrompt(
```

Comment:
We shouldn't have these static methods that are common to both services here, we should be using the ones in the separate file